src/routes/boxer.js

```diff
@@ -0,0 +1,57 @@
+const { Boxer } = require('../db');
+
```

Comment:
add :
```
express= require('express');
router = express.Router();
```
then add your routes here because this is your 'route' file for boxer. Then change the app to router. Then change your function to async function look like this _async function nameOfFunction (parameter1, parameter2){code goes here}_, because if you leave them as be you'll get errors